src/nvim/option.c

```diff
@@ -4195,6 +4255,21 @@ set_num_option (
 
   options[opt_idx].flags |= P_WAS_SET;
 
+  if (!starting && errmsg == NULL) {
+    char_u buf_old[11], buf_new[11], buf_type[7];
+    vim_snprintf((char *)buf_old, 10, "%ld", old_value);
+    vim_snprintf((char *)buf_new, 10, "%ld", value);
+    vim_snprintf((char *)buf_type, 7, "%s",
```

Comment:
ARRAY_SIZE

// Note: I am not sure, what would actually be better. Though I heard about people preferring ARRAY_SIZE I tend to use sizeof in case of `char` arrays.